Source/Core/Core/AchievementManager.cpp

```diff
@@ -270,6 +273,82 @@ void AchievementManager::ActivateDeactivateRichPresence()
       nullptr, 0);
 }
 
+void AchievementManager::FetchBadges()
+{
+  if (!m_is_runtime_initialized || !IsLoggedIn() || !Config::Get(Config::RA_BADGES_ENABLED))
+    return;
+  if (!m_player_badge.loaded)
+  {
+    m_image_queue.EmplaceItem([this] {
+      {
+        std::lock_guard lg{m_lock};
+        std::string username = Config::Get(Config::RA_USERNAME);
+        rc_api_fetch_image_request_t icon_request = {.image_name = username.c_str(),
+                                                     .image_type = RC_IMAGE_TYPE_USER};
+        if (RequestImage(icon_request, &m_player_badge) == ResponseType::SUCCESS)
+          INFO_LOG_FMT(ACHIEVEMENTS, "Successfully downloaded player badge.");
```

Comment:
I'm not sure what the protocol is for log messages so take my comments with that in mind.

Do you need the successful log messages?  Are they useful for debugging?  If not, I'd remove them.  If they are useful for debugging, they might be better off in a different category instead of 'info'.